### PR TITLE
Test resources

### PR DESCRIPTION
**Pull request by rrasouli**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 2aadd2b22cf2cd5ce5f209e4764c561db73dcbdb:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_resources.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import pytest
+from utils import testgen
+
+pytestmark = [pytest.mark.meta(server_roles='+ smartproxy + smartstate')]
```
Comment (mfalesni) [inline cfme/tests/infrastructure/test_resources.py]: This must be '+smartproxy +smartstate'

File: cfme/tests/infrastructure/test_resources.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_number_of_cpu(provider, soft_assert):
+    provider.load_details()
+    soft_assert(
+        int(provider.get_detail("Properties",
+                                "Aggregate Node CPU Resources")) < 1,
```
Comment (psav) [inline cfme/tests/infrastructure/test_resources.py]: Are you sure this assertion is correct?

**Comment by rrasouli** [comment]
Closing this PR for better tracking on newer PR's

**Pull request closed without merging**